examples/csvpb/go.mod

```
@@ -0,0 +1,11 @@
+module github.com/alpstable/gidari/examples/csvpb
+
+go 1.19
```

@Conor-Fleming Can we add this line so that we are pointing at the remove instance of Gidari? v.0.1.0 causes an error since the "Writer" has not been added to the HTTPRequest in that version.
```suggestion
go 1.19

replace github.com/alpstable/gidari => ../../

```